resource-managers/kubernetes/token-refresh-server/src/main/scala/org/apache/spark/security/kubernetes/constants.scala

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.spark.security.kubernetes
+
+package object constants {
+
+  val REFRESH_SERVER_KERBEROS_PRINCIPAL = "kimoonkim"
```

Comment:
interesting principal here ;) is this for testing?

Reply:
Yes, only for testing. We should make this configurable.

Reply:
Fixed.